js/plugin.iiif-iabookreader_strawberry.js

```diff
@@ -94,25 +109,44 @@ BookReader.prototype.setupTooltips = function() {
 
 BookReader.prototype.parseSequence = function (sequenceId) {
   var self = this;
-
-  jQuery.each(self.jsonLd.sequences, function(index, sequence) {
-    // try with a specific sequenceID
-    if (sequenceId!= null) {
-      if (sequence['@id'] === sequenceId) {
+  if(self.getApiVersion() == "3.x") {
```

Comment:
There is a `jQuery.each(self.jsonLd.sequences, function(index, sequence)` { missing here

Reply:
API 3.x does not use sequences. We iterate through jsonLd.items in `getImagesListApi3(self.jsonLd.items)`.

Reply:
Ah. Ok, I'll look at that.

Reply:
Gracias @patdunlavey !! ❤️

Reply:
I added structure ordering to getImagesListApi3. In doing so, I found that IIIF Presentation API 3.0 Manifest has a bug in the structure array for pdf documents. I think the problem is that it uses the node id in the structure range item id, rather than the cantaloupeid.
I check for this condition in getImagesListApi3 and use the original item ids in that case (and console.log some notices). I played with fixing the API 3.0 manifest, but haven't got it working yet.

Reply:
Not pretty, but I got the structure range items into the manifest.
[api3manifest-test.twig.txt](https://github.com/esmero/format_strawberryfield/files/8392098/api3manifest-test.twig.txt)
.